json/criteres-wcag-ease-fr.json

```
@@ -1005,15 +1005,15 @@
 	},
 	{
 		"themes": "Mise en page",
-		"title": "La taille des zones cliquables ou tactiles est-elle suffisante&nbsp;?",
+		"title": "Les éléments interactifs ont-ils une surface minimale de 24*24 pixels ou un espacement suffisant ?",
```

Not sure if it is still use ful or not but to be consistent:

```suggestion
		"title": "Les éléments interactifs ont-ils une surface minimale de 24*24 pixels ou un espacement suffisant&nbsp;?",
```